web/src/presence.ts

```diff
@@ -39,6 +39,8 @@ export const presence_info = new Map<number, PresenceStatus>();
 // TODO: Improve this comment.
 export let presence_last_update_id = -1;
 
+export let presence_history_limit_days_for_web_app: number | undefined;
```

Comment:
This isn't used yet, so before we merge, I need to tweak the web app code to adjust the `Last active more than X days ago` string based on this value. Just waiting for confirmation that the mechanism of passing this information to the webapp is the way we want.

Reply:
I suggested a tweak but I think it's basically fine. Only question is if we want it to be "passed" at all vs. just being hardcoded in both places agreeing on, say, a year.

Reply:
Yeah, hard-coding is another option - but then we have to make sure the two separate places don't get out of sync, so I felt the passing approach was more robust.

Fixed the `Last active .... ago` web app text now.